bats/Makefile

```diff
@@ -7,10 +7,12 @@ containers:
 	bats --show-output-of-passing-tests ./tests/containers/
 
 # https://www.shellcheck.net/wiki/SC1091 -- Not following: xxx was not specified as input (see shellcheck -x)
+# https://www.shellcheck.net/wiki/SC2001 -- See if you can use ${variable//search/replace} instead:
```

Comment:
Don't disable the rule globally; just for the single line where you need an exception:

```diff
--- bats/tests/profile/create-profile-output.bats
+++ bats/tests/profile/create-profile-output.bats
@@ -192,6 +192,7 @@ assert_check_registry_output() {
     salt=$$
     # Can't write into ...\Policies\ as non-administrator, so populate a different directory.
     safePolicyName="fakeProfile${salt}"
+    # shellcheck disable=SC2001
     sed "s/Policies/${safePolicyName}/" <<<"$mainOutput" >"${bashSideTemp}/${testFile}"
     reg.exe import "${winSideTemp}\\${testFile}"
     reg.exe delete "HKCU\\Software\\${safePolicyName}\\Rancher Desktop" /f /va
```

Reply:
so that's how it's done